json_schemas/instrument_schemas/SRS_lock_in.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$id": "json_schemas/instrument_schemas/SRS_lock_in.json",
```

Comment:
Please name this `SRS_830` instead of `SRS_lock_in` to be consistent with the file name under `devices/`